code/items/perks/Radar1.cs

```diff
@@ -0,0 +1,15 @@
+using Sandbox;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_radar1")]
+    public class Radar1 : TTTPerk, IBuyableItem
```

Comment:
What's the difference between `Radar1` vs. `Radar2`?

Reply:
Just testing the UI stuff